packages/legend-graph/src/models/protocols/pure/v1/engine/V1_EngineServerClient.ts

```diff
@@ -51,7 +51,7 @@ import type { EXECUTION_SERIALIZATION_FORMAT } from '../../../../../graphManager
 import type { V1_ExternalFormatDescription } from './externalFormat/V1_ExternalFormatDescription';
 import type { V1_ExternalFormatModelGenerationInput } from './externalFormat/V1_ExternalFormatModelGeneration';
 
-enum CORE_ENGINE_TRACER_SPAN {
+export enum V1_ENGINE_CLIENT_ACTION {
```

Comment:
uhm, the responsibility is a bit different here though, one is in the form of `tracer span: transform Pure code to protocol` vs `log event: TRANSFORM_PURE_CODE_TO_PROTOCOL`. Let's revert this change for now

Reply:
@akphi  you think it would be better to add a ENGINE_LOG_EVENT to that file instead ?

Reply:
yeah, let's do that. We should not use this, the tracer really should only be here in the network client for now.